pkg/handlers/v1/notification.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/asecurityteam/nexpose-scan-notifier/pkg/domain"
+	"github.com/asecurityteam/nexpose-scan-notifier/pkg/logs"
+)
+
+// Output contains a list of completed Nexpose scans.
+type Output struct {
+	Response []scanNotification `json:"response"`
+}
+
+// scanNotification represents
+type scanNotification struct {
+	ScanID string `json:"scanID"`
+	SiteID string `json:"siteID"`
+}
+
+// NotificationHandler takes a duration and returns a list of completed scans.
+type NotificationHandler struct {
+	ScanFetcher      domain.ScanFetcher
+	TimestampFetcher domain.TimestampFetcher
+	TimestampStorer  domain.TimestampStorer
+	Producer         domain.Producer
+	LogFn            domain.LogFn
+}
+
+// Handle accepts takes Input with a duration, queries for completed scans within the duration,
```

Comment:
Task: Seems like we're not using the duration anymore, so this comment needs to be updated.

Reply:
👍 Fixed